submissions/semenjob/index.html

```diff
@@ -0,0 +1,127 @@
+  </head>
+  <body>
+    <header class="header">
+      <ul class="nav__list">
+        <li tabindex="1" class="header__list">
+          <a class="head__link" href="#">Mail</a>
+        </li>
+        <li tabindex="1" class="header__list">
+          <a class="head__link" href="#">Images</a>
+        </li>
+      </ul>
+      <input
+        tabindex="1"
+        class="popup__checkbox"
+        type="checkbox"
+        id="popup__checkbox"
+      />
+      <label for="popup__checkbox" class="popup__checkbox__label">
+        <img
+          class="popup__icon__head"
+          src="./icon/popup-button.png"
+          alt="button logo"
+        />
+      </label>
+      <nav class="popup">
+        <ul class="popup__list">
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/apps.png" alt="Apps logo" />
+            <span class="icon_text">Apps</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/bell.png" alt="Bell logo" />
+            <span class="icon_text">Bell</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/blogs.png" alt="Blogs logo" />
+            <span class="icon_text">Blogs</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img
+              class="popup__icon"
+              src="./icon/calendar.png"
+              alt="Calendar logo"
+            />
+            <span class="icon_text">Calendar</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/chat.png" alt="Chat logo" />
+            <span class="icon_text">Chat</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img
+              class="popup__icon"
+              src="./icon/classroom.png"
+              alt="Classroom logo"
+            />
+            <span class="icon_text">Classroom</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img
+              class="popup__icon"
+              src="./icon/collection.png"
+              alt="Collection logo"
+            />
+            <span class="icon_text">Collection</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img
+              class="popup__icon"
+              src="./icon/contacts.png"
+              alt="Contacts logo"
+            />
+            <span class="icon_text">Contacts</span>
+          </li>
+          <li class="more">
+            <input
+              class="more__popup__checkbox"
+              type="checkbox"
+              id="more__popup__checkbox"
+              tabindex="2"
+            />
+            <label
+              class="more__popup__checkbox__label label__disable"
+              for="more__popup__checkbox"
+            >
+              More</label
```

Comment:
strange formatting

Reply:
this is a prettier from code formatting recommendation